crates/miden-lib/asm/miden/auth/ecdsa_k256_keccak.masm

```diff
@@ -0,0 +1,187 @@
+use.miden::active_account
+use.miden::native_account
+use.miden::auth
+use.miden::tx
+use.std::crypto::dsa::ecdsa::secp256k1
+
+# CONSTANTS
+# =================================================================================================
+
+# The event to request an authentication signature.
+const.AUTH_REQUEST_EVENT=event("miden::auth::request")
+
+# The slot in this component's storage layout where the public key is stored.
+const.PUBLIC_KEY_SLOT=0
+
+# Local Memory Addresses for multisig operations
+const.NUM_OF_APPROVERS_LOC=0
```

Comment:
Not from this PR, but I'm not seeing `NUM_OF_APPROVERS_LOC`. Maybe worth creating an issue to remove it. cc @partylikeits1983.